app/uk/gov/hmrc/thirdpartyapplication/domain/models/ApplicationUpdate.scala

```diff
@@ -45,6 +48,7 @@ trait GatekeeperSpecificApplicationUpdate extends ApplicationUpdate {
 }
 case class ChangeProductionApplicationName(instigator: UserId, timestamp: LocalDateTime, gatekeeperUser: String, newName: String) extends GatekeeperSpecificApplicationUpdate
 case class DeclineApplicationApprovalRequest(gatekeeperUser: String, reasons: String, timestamp: LocalDateTime) extends GatekeeperSpecificApplicationUpdate
+case class DeleteApplicationByGatekeeper(actor: Actor, requestedByEmailAddress: String, reasons: String, timestamp: LocalDateTime) extends ApplicationUpdate
 
 
 trait ApplicationUpdateFormatters {
```

Comment:
this list is getting long - probably time to "enum" the types - see api-platform-application-events for an example of not using "public" or "enum"